### ISSUE TITLE
TensorRT int8 engine (convert from qat onnx using pytorch-quantization)  produce wrong inference result.

### ISSUE THREAD

**Issue by Real-YeJ** [NVIDIA/TensorRT#2553]
## Description
I trained a qat model for object detection using pytorch-quantization follow the user guide:  (https://docs.nvidia.com/deeplearning/tensorrt/pytorch-quantization-toolkit/docs/tutorials/quant_resnet50.html)

Entropy and percentile calibration methods are used during the training.  
Histogram based calibration for activations and the default max calibration for weights. 

Everything seems fine during the qat finetune. However, when I export the qat pytorch model to onnx model with qdq op, and convert the onnx model to int8 trt, there shows a convert warning:
<img width="1521" alt="截屏2022-12-15 11 28 04 2" src="https://user-images.githubusercontent.com/37889627/208229917-10650481-35fd-4e06-aab7-876d59808a8a.png">

and the int8 trt engine  inference  wrong with empty result. 

The quantized scale of histogram based calibration is much smaller than max calibration which works  normally but has low accuracy.

can anyone provide some suggestions, thx

**Comment by zerollzeng** [comment]
@ttyio ^ ^

**Comment by Real-YeJ** [comment]
I have solved this problem. it seems a normal warning when converting qat model using entropy calibration to int8 trt.
